wgsl/index.bs

```diff
@@ -9754,6 +9754,17 @@ There is no risk of being trapped in a cycle, as recurrence is forbidden in the
 
 Note: Another way of saying the same thing is that we do a topological sort of functions ordered by the "is a (possibly indirect) callee of" partial order, and analyze them in that order.
 
+Additionally, the analysis computes and propagates locations of [=call sites=] for [=builtin functions that compute a derivative=].
+This is used to [=triggered|trigger=] and filter [=diagnostics=] for [=uniformity failures=] on those calls.
+A set of such source locations is called a <dfn noexport>witness set</dfn>.
+Witness sets only support adding elements, not removing elements.
+In particular, a witness set is only formed via the following operations:
+* Create an empty set.
+* Create a witness set containing a single source location.
+* Form a witness set from the union of two others.
+
+Witness sets can be implemented such that their total cost across the entire uniformity analysis is linear in the source program size.
```

Comment:
If you implement with a persistent trees, in the sense of https://en.wikipedia.org/wiki/Persistent_data_structure then  you get this.

Reply:
Remove this sentence per feedback form @jimblandy